relay/writer/file.go

```diff
@@ -332,23 +332,28 @@ func (w *FileWriter) handleDuplicateEventsExist(ev *replication.BinlogEvent) (*R
 // 2.
 //    a. update the position with the event's position if the transaction finished
 //    b. update the GTID set with the event's GTID if the transaction finished
-// 3. truncate any un-finished events/transactions
+// 3. truncate any uncompleted events/transactions
```

Comment:
"unfinished" is fine, or use "incomplete", but "uncompleted" is definitely wrong.